Project.toml

```
@@ -1,15 +1,22 @@
 name = "CurveFit"
 uuid = "5a033b19-8c74-5913-a970-47c3779ef25c"
 authors = ["Paulo José Saiz Jabardo <pjabardo@gmail.com> and contributors"]
-version = "0.6.1"
+version = "0.7.0"
```

```suggestion
version = "1.0.0"
```

Let's just make it the 1.0